social.md

```diff
@@ -76,7 +76,7 @@ The library must implement provider callbacks for all social provider
 directories that are mapped to the specified Stormpath application.  The
 callback URLs for each provider should take the form of:
 
-> `/<stormpath.web.socialProviders.callbackRoot>/<providerId>`
+> `<stormpath.web.socialProviders.[providerId].callbackUri>/<providerId>`
```

Comment:
IIUC, different callback uris for each provider is necessary until we become the intermediary to automate this.  Once we are the intermediary, the app only needs a single callback for all of our assertions (id site, oauth, saml), because at that point, the behavior to perform would be represented in the JWT assertion rather than by which URL in the app where we redirect the user.  A single URL is also nicer when configuring your security framework and logging, etc.

At that time, I suppose our config (c/w)ould look like the following:

```yaml
  social:
    uri:  "/sso"
    facebook:
      scope: "email"
    github:
      scope: "user:email"
    # etc ...
````

Thoughts?